pyprophet/cli/score.py

```diff
@@ -43,7 +43,7 @@
     "--classifier",
     default="LDA",
     show_default=True,
-    type=click.Choice(["LDA", "SVM", "XGBoost"]),
+    type=click.Choice(["LDA", "SVM", "XGBoost", "LDA_XGBoost"]),
```

Comment:
Can you update the `RunnerConfig` in the `_config.py` with the added classifier.